CMakeLists.txt

```diff
@@ -41,6 +41,9 @@ file(GLOB AWS_COMMON_HEADERS
 file (GLOB AWS_COMMON_EXTERNAL_HEADERS
         "include/aws/common/external/*.h")
 
+file (GLOB AWS_COMMON_EXTERNAL_INSTALLED_HEADERS
```

Comment:
If you have any suggestions for a better name for this variable, please feel free to share them.

Reply:
I like it